src/app/core/entity/schema-datatypes/array.datatype.spec.ts

```diff
@@ -19,18 +17,12 @@ describe("Schema data type: array", () => {
 
   beforeEach(waitForAsync(() => {
     TestBed.configureTestingModule({
-      imports: [CoreModule],
-      providers: [
-        ComponentRegistry,
-        {
-          provide: DefaultDatatype,
-          useValue: new ConfigurableEnumDatatype({
-            getEnumValues: () => defaultInteractionTypes,
-          } as any),
-          multi: true,
-        },
-      ],
+      imports: [MockedTestingModule.withState()],
```

Comment:
in the spirit of "unit tests" trying to minimize the external dependency of the test, I had intentionally put up this test setup. But if the MockedTestingModule helps avoid further problems, that's also cool.

Reply:
Yes, unfortunately to whole datatype dependencies would require a lot of mocking